package.json

```diff
@@ -22,8 +22,7 @@
   },
   "repository": "https://github.com/emberjs/ember-inspector",
   "engines": {
-    "node": ">= 0.10.0",
-    "npm": ">= 2.0.0"
+    "node": ">= 0.10.0"
   },
   "emberVersionsSupported": [
```

Comment:
can you explain what this key and the next one do? Is it supposed to be an empty string in the list?